src/app/(homepage)/Demo.js

```diff
@@ -2,30 +2,38 @@ import OosStepper from "@/components/OosStepper";
 import { Box, Container } from "@mui/material";
 import Typography from "@mui/material/Typography";
 
-function DemoHeader({sx}) {
+
+function DemoHeader({ sx, subheading1, heading1, heading2}) {
```

Comment:
Heading 2 is unused here, why passing it at all. It should be highlighted by VScode.